src/components/CollectionItem.js

```diff
@@ -17,14 +17,8 @@ export const PositionedImageContainer = styled(ImageContainer)`
   left: 30px;
 `;
 
-export default function CollectionItem({ className, onClick }) {
-  return (
-    <ImageContainer
-      classname={className}
-      onClick={onClick}
-      src="https://d2k4q26owzy373.cloudfront.net/700x700/games/uploaded/1559254200327-61EFZADvURL.jpg"
-    />
-  );
+export default function CollectionItem({ className, onClick, src }) {
+  return <ImageContainer classname={className} onClick={onClick} src={src} />;
```

Comment:
I think, in order for it to work, you need to use "className=" instead of "classname=".
onClick - normally - is the event. Functions that are supposed to be be executed after the event occurs should be named "handle..." so that you know what will happen after the event.